### PR TITLE
Black Capped Chickadee

### PR DESCRIPTION
**Pull request by hashimhassany**
This for this issue!

https://github.com/instructlab/taxonomy/issues/844

### CODE REVIEW

**Comment by jjasghar** [comment]
@hashimhassany thanks for this! Can you fix the linting issue?

**Comment by hashimhassany** [comment]
@jjasghar File name changed to fix linting issue!

File: knowledge/animals/birds/black_capped_checkadee/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+created_by: Hashim Hassany
```
Comment (jjasghar) [inline knowledge/animals/birds/black_capped_checkadee/qna.yaml:1]: Please add your GitHub username here.

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix the created_by then we can run the linter.

File: knowledge/animals/birds/black_capped_checkadee/qna.yaml (hunk 1 above)
Comment (hashimhassany) [reply knowledge/animals/birds/black_capped_checkadee/qna.yaml]: I have changed this to my GitHub user name.

**Comment by jjasghar** [comment]
Please fix the linting issues.

**Comment by jjasghar** [comment]
@instructlab-bot precheck

File: knowledge/animals/birds/black_capped_checkadee/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+created_by: hashimhassany
+domain: animals
+seed_examples:
+- answer: North America
+  question: Where does the Black-capped chickadee originate.
+- answer: Paridae family
+  question: This bird is the member of which family?
```
Comment (jjasghar) [inline knowledge/animals/birds/black_capped_checkadee/qna.yaml:7]: This isn't a clear enough question, it should be "What is the black-capped chickadee membership of which family" or the like.

File: knowledge/animals/birds/black_capped_checkadee/qna.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+- answer: 6-8 eggs on average
+  question: How many eggs does this bird lay on average?
```
Comment (jjasghar) [inline knowledge/animals/birds/black_capped_checkadee/qna.yaml:9]: "this bird" needs to be more focused.

File: knowledge/animals/birds/black_capped_checkadee/qna.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+- answer: Once a year
+  question: How many times a year does this bird molt?
```
Comment (jjasghar) [inline knowledge/animals/birds/black_capped_checkadee/qna.yaml:11]: ditto.

File: knowledge/animals/birds/black_capped_checkadee/qna.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+- answer: Molt lasts two to three months
+  question: How long does a molt last?
```
Comment (jjasghar) [inline knowledge/animals/birds/black_capped_checkadee/qna.yaml:13]: This needs a more direct reference to this bird.

**Review by jjasghar — CHANGES_REQUESTED** [review]
This is a great start, but the model needs more focus to help get the correct answers. Please update your qna with these.

**Comment by jjasghar** [comment]
Hi! :wave:
With the new update and release of 0.18.0, our [base knowledge submissions](https://github.com/instructlab/taxonomy/?tab=readme-ov-file#knowledge-yaml-examples) have drastically changed. This has given us an opportunity to significantly improve the accuracy of our fine-tuned models, which is a major win for our community!

But alas, that has caused some changes that this PR isn't able to continue forward. We greatly appreciate the work you've put into this, and wish we could accept it but the amount of work to get it to be `version: 3` is unrealistic for us to ask.
If you _do_ want to do this, please don't hesitate to take the `qna.yaml` update it, and build it out. Otherwise, we hope you come back to our community with a new submission and help build out our Open Source community-driven LLM.

If you have any questions directly, please don't hesitate to comment on the PR or email me at [awesomeATinstructlab.ai](http://awesomeatinstructlab.ai/) (yes, really), and I'll do my best to answer your questions.

**Pull request closed without merging**